core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -443,6 +443,40 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  val PARALLEL_PARTITION_DISCOVERY_THRESHOLD =
+    ConfigBuilder("spark.rdd.sources.parallelPartitionDiscovery.threshold")
+      .doc("The maximum number of paths allowed for listing files at driver side. If the number " +
+        "of detected paths exceeds this value during partition discovery, it tries to list the " +
+        "files with another Spark distributed job. This configuration is effective only when " +
+        "using file-based using HadoopFSUtils")
+      .version("1.5.0")
```

Comment:
This configuration looks new. Not in 1.5.0.

Reply:
BTW, where are these configurations used?

Reply:
Oops will fix. Yes these are new core configurations that copied from [SQLConf](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L990). I don't have any better idea on how to solve the duplication yet.